.travis.yml

```diff
@@ -16,6 +16,7 @@ matrix:
 
 before_install:
   - sudo apt-get update -qq
+  - sudo apt-get install hwdata -y
```

Comment:
nit: similar to the above apt-get update command, id use -q flag as then apt-get output is more suitable for logging

i.e replace `-y` with `-yq`

```
-q, --quiet
Quiet. Produces output suitable for logging, omitting progress indicators. More q's will produce more quiet up to a maximum of two. You can also use -q=# to set the quiet level, overriding the configuration file. Note that quiet level 2 implies -y, you should never use -qq without a no-action modifier such as -d, --print-uris or -s as APT may decided to do something you did not expect.
```